server/app/com/xsn/explorer/services/BlockService.scala

```diff
@@ -122,6 +122,25 @@ class BlockService @Inject()(
     result.toFuture
   }
 
+  def getBlockRewards(height: Height): FutureApplicationResult[Option[BlockRewards]] = {
```

Comment:
This makes no sense because you already have the full block and you are basically retrieving it again.